installing/installing_ibm_cloud_public/user-managed-encryption-ibm-cloud.adoc

```diff
@@ -0,0 +1,42 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="user-managed-encryption-ibm-cloud"]
+= User-managed encryption for {ibm-cloud-title}
+include::_attributes/common-attributes.adoc[]
+:context: user-managed-encryption-ibm-cloud
+
+toc::[]
+
+By default, provider-managed encryption is used to secure the following when you deploy an {product-title} cluster:
+
+* The root (boot) volume of control plane and compute machines.
```

Comment:
These list items do not need a period since they aren't complete sentences per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists).